doc/source/tune/tutorials/tune-distributed.rst

```diff
@@ -166,7 +166,7 @@ In GCP, you can use the following configuration modification:
         scheduling:
           - preemptible: true
 
-Spot instances may be removed suddenly while trials are still running. Often times this may be difficult to deal with when using other distributed hyperparameter optimization frameworks. Tune allows users to mitigate the effects of this by preserving the progress of your model training through :ref:`checkpointing <tune-function-checkpointing>`.
+Spot instances may be removed suddenly while trials are still running. Often times this may be difficult to deal with when using other distributed hyperparameter optimization frameworks. Tune allows users to mitigate the effects of this by preserving the progress of your model training through :ref:`checkpointing <tune-trial-checkpoint>`.
```

Comment:
Actually removed the sentence "This can cause problems when using ..."

Feels like we are saying other non-ray-tune choices are not good, without being really specific about it.